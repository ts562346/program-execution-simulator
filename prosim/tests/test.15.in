test1 6
LOOP 4
  LOOP 3
  DOOP 1
  END
END
HALT

