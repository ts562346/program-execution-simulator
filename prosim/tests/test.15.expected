00000: DOOP
00001: DOOP
00002: DOOP
00003: DOOP
00004: DOOP
00005: DOOP
00006: DOOP
00007: DOOP
00008: DOOP
00009: DOOP
00010: DOOP
00011: DOOP
00012: HALT

DOOP count : 12
DOOP time  : 12
BLOCK count: 0
BLOCK time : 0
